app/models/outgoing_message.rb

```diff
@@ -470,6 +470,11 @@ def template_changed
     end
   end
 
+  def regex_escape(content)
+    text = content.gsub("\r", "\n") #shouldn't happen, but just in case
```

Comment:
as it's technically possible that it's carriage returns all the way down, I thought this was safer